org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/resource/ApiExposedFilter.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.rest.resource;
+
+import java.io.IOException;
+import java.net.URI;
+
+import jakarta.annotation.Priority;
+import jakarta.ws.rs.Priorities;
+import jakarta.ws.rs.container.ContainerRequestContext;
+import jakarta.ws.rs.container.ContainerRequestFilter;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriInfo;
+
+import org.eclipse.scout.rt.api.data.ApiExposed;
+import org.eclipse.scout.rt.platform.ApplicationScoped;
+import org.eclipse.scout.rt.platform.BEANS;
+import org.eclipse.scout.rt.platform.Platform;
+import org.eclipse.scout.rt.rest.error.ErrorResponseBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+
+@ApplicationScoped
+@Priority(Priorities.AUTHORIZATION)
```

Comment:
switch the two annotations (e.g. order by length(annotation))

Reply:
https://github.com/eclipse-scout/scout.rt/compare/6afdcd507e7cd4fef7c7ed815fdbd73c632c1996..a3cc68288dc0cb782cd10418693764f45cc3fd31